cytosnake/common/errors.py

```diff
@@ -72,3 +79,75 @@ class WorkflowNotFoundError(BaseFileNotFound):
 
 class WorkflowFailedException(BaseWorkflowException):
     """Raised if a workflow fails during runtime"""
+
+
+class ProjectExistsError(BaseFileExistsError):
+    """Raised when `.cytosnake` file is found within a directory indicating
+    that the current directory has already been set up for cytosnake analysis"""
+
+
+# -----------------------
+# Error handling functions
+# -----------------------
+def display_error(
```

Comment:
The purpose of `display_error` is to ensure that the CLI tool exits non-zero exit status. It also allows to edit exception messages if needed in order to make error messages more understandable. So in the future, if CytoSnake contains it's own error code, the display error will be extremely helpful. 

How do you see `Logger.error()` play a role with `display_error`

Reply:
I'd recommend looking into how [`logging`](https://docs.python.org/3/library/logging.html) might be used with some of this work in order to take advantage of built-in capabilities. Doing this may standardize how log messages are displayed or stored. I bring this up as a suggestion towards increasing development velocity.

Reply:
Will definitely take you up on that! Thanks!